scalafix-tests/unit/src/test/scala/scalafix/tests/interfaces/ScalafixArgumentsSuite.scala

```diff
@@ -461,20 +471,15 @@ class ScalafixArgumentsSuite extends AnyFunSuite with DiffAssertions {
 
     val scalacOptions = Array[String](
       "-Xsource:3",
-      "-Yrangepos",
       removeUnused,
-      s"-Xplugin:${SemanticdbPlugin.semanticdbPluginPath()}",
-      "-Xplugin-require:semanticdb",
       "-classpath",
       s"${scalaLibrary.mkString(":")}",
-      s"-P:semanticdb:sourceroot:$src",
-      s"-P:semanticdb:targetroot:$target",
       "-d",
       d.toString,
       main.toString
-    )
+    ) ++ CompatSemanticdb.scalacOptions(src, target)
```

Comment:
should `specificScalacOption2` be added here too?

Reply:
This does not need `removeUnused` so no. This test will be removed with https://github.com/scalacenter/scalafix/pull/1666.